apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -3108,4 +3108,27 @@ describe('Meteor.methods', function () {
 			});
 		});
 	});
+
+	describe('[@saveSettings]', () => {
+		it('should return an error when trying to save a "NaN" value', () => {
```

Comment:
Can we add other test cases? e.g. boolean, strings, null, undefined, and finally one ensuring we can save a number (int, float) value.